spec/adapter/pg_uuid_spec.cr

```diff
@@ -0,0 +1,436 @@
+require "./spec_helper"
+require "../src/adapter/pg"
+
+class Parent < Granite::ORM::Base
```

Comment:
We are changing the way we write specs.  Instead of having 3 separate specs (4 with this) per adapter, we are looking to write the specs per feature.

Take a look at the new query spec.  Can you look at moving this over to that style?  Call the spec something like `uuid_primary_key_spec.cr` or something like that and focus on just testing the functionality needed to support a uuid.

https://github.com/amberframework/granite-orm/blob/master/spec/granite_orm/querying_spec.cr

Reply:
I have looked at these specs, they are very hard to read, use macro or not use macro, this is a question.